src/main/java/org/spongepowered/common/registry/builtin/sponge/MapColorTypeStreamGenerator.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.registry.builtin.sponge;
+
+import net.minecraft.block.material.MaterialColor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.spongepowered.api.ResourceKey;
+import org.spongepowered.api.map.color.MapColorType;
+import org.spongepowered.common.map.color.SpongeMapColorType;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public class MapColorTypeStreamGenerator {
+	private static final Map<Integer, MapColorType> COLOR_INDEX_MAP = new HashMap<>();
+	private static final Logger logger = LogManager.getLogger(MapColorTypeStreamGenerator.class.getSimpleName());
+
+	public static Stream<MapColorType> stream() {
+		///////////////////////// ORDER MATTERS HERE ////////////////////
+		return Stream.of(
```

Comment:
Correct the formatting here please to match the others.

Reply:
Not sure exactly what you want me to do here.